pypesto/objective/aggregated.py

```diff
@@ -113,6 +116,9 @@ def call_unprocessed(
                 "The length of `kwargs_list` must match the number of "
                 "objectives you are aggregating."
             )
+        for objective_, objective_kwargs in zip(self._objectives, kwargs_list):
+            if objective_.share_return_dict:
+                objective_kwargs["return_dict"] = return_dict
```

Comment:
Same as in the base objective (see above or below comment)

Reply:
I would actually argue that the outer `return_dict` should take precedence over inner `return_dict` and that we don't need the `if objective_.share_return_dict:` check at all.